packages/utils/package.json

```diff
@@ -36,15 +62,17 @@
     "test:browsers": "yarn karma start karma.config.cjs",
     "check-readme": "typescript-docs-verifier"
   },
-  "types": "lib/index.d.ts",
   "dependencies": {
     "@chainsafe/as-sha256": "^0.3.1",
     "any-signal": "2.1.2",
     "bigint-buffer": "^1.1.5",
     "case": "^1.6.3",
     "chalk": "^4.1.2",
     "js-yaml": "^3.13.1",
-    "winston": "^3.3.3"
+    "winston": "^3.3.3",
+    "prom-client": "^14.1.0",
+    "cross-fetch": "^3.1.4",
+    "jwt-simple": "0.5.6"
```

Comment:
jwt is node only dependency, please review the source of the dependency before including in utils. Utils must be consumable from browser code, so this line will break

https://github.com/hokaccha/node-jwt-simple/blob/a812607fa498e147a0816cbbbbf386df60d9fd6c/lib/jwt.js#L13